.github/common.env

```diff
@@ -1,6 +1,6 @@
 # Shared common variables
 
-CI_IMAGE_VERSION=master-862722333
+CI_IMAGE_VERSION=master-1091773482
 CI_TOXENV_MAIN=py37,py38,py39,py310,py311
 CI_TOXENV_PLUGINS=py37-plugins,py38-plugins,py39-plugins,py310-plugins,py311-plugins
```

Comment:
I think you need to add py312 to the above two lines to make it run in CI (I wonder why we have this)

Reply:
Fixed.